packages/tree-select/package.json

```diff
@@ -27,8 +28,8 @@
 		"src"
 	],
 	"scripts": {
-		"clean": "npx rimraf dist",
+		"clean": "npx rimraf dist types",
 		"prepublish": "npm run clean",
-		"prepare": "transpile"
+		"prepare": "tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json"
```

Comment:
is `tsc` multicore capable? would it help to run these in parallel?

Reply:
looks like no https://github.com/Microsoft/TypeScript/issues/30235

Reply:
Indeed! I'll keep that in mind when I get back to this.